Assignment/solution_student/css/style.css

```diff
@@ -0,0 +1,138 @@
+/*GLOBAL STLES*/
+
+html {
```

Comment:
I would recommend applying these styles to the `body` element instead of the `html` element — it's not necessarily incorrect, but it just makes more sense to keep the highest-level element that styled the `body` element, since it is the first element that appears in its entirety in the viewport.